kong/init.lua

```diff
@@ -755,7 +772,7 @@ end
 
 
 function Kong.exit_worker()
-  if kong.configuration.role ~= "control_plane" and ngx.worker.id() == 0 then
+  if process.type ~= "privileged agent" and kong.configuration.role ~= "control_plane" and worker_id() == 0 then
```

Comment:
should use `not is_dp_worker_process()`

Reply:
@chronolaw I tried it, requiring "kong.clustering.utils"  causes errors on `init.lua`.

So either we need to move it else where or just keep it like it is.

Reply:
OK, thanks.